articles/azure-monitor/app/opencensus-python.md

```diff
@@ -142,8 +311,21 @@ First you have to create an Application Insights resource, which will generate a
 
      ![Screenshot of end-to-end transaction interface](./media/opencensus-python/0009-end-to-end-transaction.png)
 
+## View your data with queries
```

Comment:
Add note to link here if customers want to learn how to query their logs:
https://docs.microsoft.com/en-us/azure/azure-monitor/platform/data-platform-logs